### PR TITLE
Reviewed virtual installation process

### PR DESCRIPTION
**Pull request by baudbox**
First PR. I follow this step for creating virtualenv. Test and working

### CODE REVIEW

**Comment by gcarq** [comment]
Yes, `venv` is part of the standard library since python 3.3 and it should be preferred if you don't need backwards compatibility. `virtualenv` uses some internal quirks (i.e. it requires a copy of the python interpreter to work properly, etc...).

File: README.md [hunk 1]
```diff
@@ -74,14 +74,20 @@ Use your favorite AUR helper and install `python-freqtrade-git`.
...
-```
-$ cd freqtrade/
-# copy example config. Dont forget to insert your api keys
+```bash
+$ mkdir freqtrade && cd freqtrade/
+# Create an isolated Python virtual environment
+$ pip install virtualenv
+$ virtualenv ./virtualenv --python=$(which python3)
+# Activate the virtualenv
+$ . virtualenv/bin/activate
```
Comment (gcarq) [inline README.md]: consider adding virtualenv to .gitignore

**Comment by vertti** [comment]
Because we require 3.6, I don't see any benefit for this change which just means more installing and more dependencies.

```
venv is a package shipped with Python 3, which you can run using python3 -m venv (although for some reason some distros separate it out into a separate distro package, such as  python3-venv on Ubuntu/Debian). It serves a similar purpose to virtualenv, and works in a very similar way, but it doesn't need to copy Python binaries around (except on Windows). Use this if you don't need to support Python 2.
```

Was there other reasons for the changes @baudbox ?

**Comment by baudbox** [comment]
Well,  you're right. I agree. I mix up the installation process.
Moreover I mess up the branch and subsequence PR. Sorry about this. This PR can be definitively rejected.

**Pull request closed without merging**